### DISCUSSION TITLE
Filament Table: Add icon in table Header

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#12208] [Help]
### Package

Table builder

### Package Version

v3.2.62

### How can we help you?

I want to add a custom `->icon()` in `$table` Header.

**Method Filament\Tables\Table::icon does not exist.**

```
    public function table(Table $table): Table
    {
        return $table
            ->icon('heroicon-o-search')
            ->heading('Table Header')
            ->description(__('Lorem Ipsum is simply dummy text.'))
            ->columns([
                // ...
            ])
            ->filters([
                // ...
            ])
            ->actions([
                // ...
            ])
            ->bulkActions([
                // ...
            ]);
    }
```

**Comment by micraux** [comment]
Hello,

You may want to create a custom view for your table header:
[https://filamentphp.com/docs/3.x/tables/advanced#customizing-the-table-header](https://filamentphp.com/docs/3.x/tables/advanced#customizing-the-table-header)

**Reply by devhoussam1998** [reply]
Thanks I already submit a PR about add icon in table header

**Reply by micraux** [reply]
Very good idea!
